lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Model for Course Programs.
+ */
+(function (define) {
+    'use strict';
+    define([
+            'backbone'
+        ], 
+        function (Backbone) {
+        return Backbone.Model.extend({
+            initialize: function(data) {
+                if (data){
+                    this.context = data;
+                    //we should populate our model by looking at the run_modes
+                    if (data.run_modes.length > 0){
+                        //We only have 1 run mode for this program
+                        this.setActiveRunMode(data.run_modes[0]);
```

Comment:
I know. This is the area where we need to do work on for ECOM-3206. I added the most dumb logic here to just get the rendering working